.github/workflows/ci.yml

```diff
@@ -82,27 +82,15 @@ jobs:
       profile: ${{ inputs.profile || 'debug' }}
       stage: ${{ inputs.stage || 'debug' }}
 
-  build-base-perf-artifacts:
```

Comment:
With the use of bencher, we no longer need to run the perf tests for `main` on every branch. Instead, the latest results of `main` will be collected and reported as CI runs on `main`.

Reply:
This is awesome, good cleanup